lfs/setup.go

```diff
@@ -28,11 +28,13 @@ var (
 		Properties: map[string]string{
 			"clean":    "git-lfs clean -- %f",
 			"smudge":   "git-lfs smudge -- %f",
+			"process":  "git-lfs filter-process",
 			"required": "true",
 		},
 		Upgradeables: map[string][]string{
-			"clean":  []string{"git-lfs clean %f"},
-			"smudge": []string{"git-lfs smudge %f"},
+			"clean":   []string{"git-lfs clean %f"},
+			"smudge":  []string{"git-lfs smudge %f"},
+			"process": []string{"git-lfs filter"},
```

Comment:
Good question. This is in the `Upgradeables` section of the setup code, meaning it's legit to overwrite this value so long as it matches one of the values in that slice. Since there was code publicly available during the course of this PR that referred to `git-lfs filter` instead of `git-lfs filter-process`, I'm going to keep this.